docs/guide/http_cache.md

```diff
@@ -501,6 +501,118 @@ set req.http.X-User-Hash = "b1731d46b0e7a375a5b024e950fdb8d49dd25af85a5c7dd5116a
 
 5\. Restart the Varnish server.
 
+##### New anonymous X-User-Hash based on `anonymous_user_id` setting
+
+Since you can configure anonymous user per SiteAccess, you need to change the default `.vcl` template to be able to benefit from this functionality. This example assumes that your SiteAccesses are matched using `URLElement` matcher. It works also with other SiteAccess matchers. As Varnish is not SiteAccess-aware itself, it is recommended to set also different session cookie names per SiteAccess.
```

Comment:
Maybe make it clear it's in cases where site access is matched by url we need to do this. In case of matching by host or port I think Varnish will keep the cache items separated.

Reply:
Also, why do we suggest using different session cookie name btw?

Reply:
Updated the part about matchers.

> Also, why do we suggest using different session cookie name btw?
I kept it from the customer's solution, but TBH I can see now, that this is not necessary. Therefore, I've removed it also.